apps/meteor/ee/app/livechat-enterprise/server/business-hour/Helper.ts

```diff
@@ -40,6 +40,7 @@ const getAgentIdsToHandle = async (businessHour: Record<string, any>): Promise<s
 export const openBusinessHour = async (businessHour: Record<string, any>): Promise<void> => {
 	const agentIds: string[] = await getAgentIdsToHandle(businessHour);
 	await Users.addBusinessHourByAgentIds(agentIds, businessHour._id);
+	await Users.makeAgentsWithinBusinessHourAvailable(agentIds);
 	await Users.updateLivechatStatusBasedOnBusinessHours();
```

Comment:
Wondering if we still need this query now? It is supposed to turn off agent's status if no BH are found active for them but considering this is openBH method, I'm not sure we need to make agent's offline here. WDYT?